src/main/java/com/microsoft/sqlserver/jdbc/SQLServerDatabaseMetaData.java

```diff
@@ -1015,15 +1015,19 @@ public int getMaxColumnsInTable() throws SQLServerException {
     public int getMaxConnections() throws SQLException, SQLTimeoutException {
         checkClosed();
         try {
-            String s = "sp_configure 'user connections'";
-            SQLServerResultSet rs = getResultSetFromInternalQueries(null, s);
-            if (!rs.next())
-                return 0;
+            SQLServerResultSet rs = getResultSetFromInternalQueries(null,
```

Comment:
If the user doesn't have required privileges, won't this query result in an exception?